src/Himali_Assignment_29/findSecMaxNum.java

```diff
@@ -0,0 +1,30 @@
+//Find second maximum from array without sorting the array
+
+package Himali_Assignment_29;
+
+public class findSecMaxNum {
+	void findSecMaxNumFromArray(int[] array){
+		int firstMax=0;
+		int secondMax=0;
+		
+			for(int i=0;i<=array[i];i++) {
+				if(array[i]>firstMax) {
+					secondMax=firstMax;
+					firstMax=array[i];
+					
+				}
```

Comment:
your code will work for this scenario only 
check this input & test your code
3, 4, 5, 6, 32, 1, 4, 69, 90